src/.env.sample

```
@@ -105,9 +105,15 @@ SCHEDULER_SERVICE_BASE_URL= '/scheduler/'
 # Default organisation code
 DEFAULT_ORGANISATION_CODE= 'sl'
 
+# Default organization code for migration
+DEFAULT_ORG_CODE = 'sl'
```

dotenv-linter (3.3.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)